src/packets/LoginUdp/LoginUdp_11/loginpackets.ts

```diff
@@ -377,6 +377,28 @@ const packets: PacketStructures = [
         { name: "message", type: "string", defaultValue: "" }
       ]
     }
+  ],
+  [
+    "H1emu.HadesInit",
```

Comment:
Hades ?

Reply:
That's what Jamie named the asset validation stuff, HadesInit isn't really used rn so we just use HadesQuery to start the validation as soon as a client connects to the login.